src/main/java/pl/touk/sputnik/configuration/CliOption.java

```diff
@@ -9,7 +9,8 @@
     CONF("cli.conf", "Configuration properties file", null),
     CHANGE_ID("cli.changeId", "Gerrit change id", null),
     REVISION_ID("cli.revisionId", "Gerrit revision id", null),
-    PULL_REQUEST_ID("cli.pullRequestId", "Stash pull request id", null);
+    PULL_REQUEST_ID("cli.pullRequestId", "Stash pull request id", null),
+    BUILD_TOOL("cli.buildTool", "Project build tool: maven or gradle", BuildTool.MAVEN.getName());
```

Comment:
I think that build tool does not change between commits or pull requests. It stays constant within project. It should an option, but not a command line one.